### PR TITLE
improved postprocessing

### PR DESCRIPTION
**Pull request by BenPortner**
Referencing #12 
1. Added tests to check whether coordinates are in German sea territory. Wrongly located wind parks far outside of German territory (e.g. in Africa) are now detected.
2. Corrected several outlier coordinates using OPSD data. Ca. 1133 coordinates corrected.

### CODE REVIEW

File: postprocessing/OEP_MaStR_cleansing.ipynb [hunk 1]
```diff
@@ -36,14 +36,21 @@
    },
```
Comment (Bachibouzouk) [inline postprocessing/OEP_MaStR_cleansing.ipynb]: @BenPortner - is it necessary to have the outputs of the notebook in the repo? If not, could you push a version without the outputs?

**Review by Bachibouzouk — APPROVED** [review]
Thanks a lot @BenPortner !

Could you also add the packages required to run your notebook to the `requirements.txt` files of the repository?

File: postprocessing/OEP_MaStR_cleansing.ipynb (hunk 1 above)
Comment (BenPortner) [reply postprocessing/OEP_MaStR_cleansing.ipynb]: Hi @Bachibouzouk! The branch used for the PR should not contain any outputs. Let me know if I'm wrong

Comment (BenPortner) [reply postprocessing/OEP_MaStR_cleansing.ipynb]: Or did you mean the notebook output cells? Do you want me to remove them all or only the graphics?

Comment (Bachibouzouk) [reply postprocessing/OEP_MaStR_cleansing.ipynb]: Yes I meant the notebook output cells ^^

I trust you to know what should not be removed. If the purpose of this notebook is mainly to be used locally by other users, I would remove the notebook's output cells.

Comment (BenPortner) [reply postprocessing/OEP_MaStR_cleansing.ipynb]: I'll remove the graphics to make the notebook smaller. I'll keep the df heads and numeral outputs. Sound good?

Comment (BenPortner) [reply postprocessing/OEP_MaStR_cleansing.ipynb]: > Thanks a lot @BenPortner !
>
>Could you also add the packages required to run your notebook to the requirements.txt files of the repository?

I'm using conda with channel conda-forge. Not sure it harmonizes well with the requirements.txt. I'll add an environment.yml instead. Will that work?

Comment (BenPortner) [reply postprocessing/OEP_MaStR_cleansing.ipynb]: Decided to remove all outputs after all!

Comment (Bachibouzouk) [reply postprocessing/OEP_MaStR_cleansing.ipynb]: I approved the PR, so you can merge whenever you want to :)

Comment (BenPortner) [reply postprocessing/OEP_MaStR_cleansing.ipynb]: I would like to merge, but I think I have no write access...

> Only those with write access to this repository can merge pull requests.

**Comment by BenPortner** [comment]
Closes #12

**Pull request merged**